packit/cli/propose_downstream.py

```diff
@@ -112,6 +114,8 @@ def propose_downstream(
     api = get_packit_api(
         config=config, dist_git_path=dist_git_path, local_project=path_or_url
     )
+    if pr is None:
+        pr = api.package_config.create_pr
```

Comment:
```suggestion
    pr = pr or api.package_config.create_pr
```

Reply:
I am worried that's not what Maťo wants here... (We would like to be able to override any value from the config.)

Reply:
Ups. You are right.